### ISSUE TITLE
URL links in ontology lookup fields

### ISSUE THREAD

**Issue by pinarpink** [elixir-luxembourg/daisy#3]
For the **Study Features**, **Disease Terms**, **Gene Terms** and **Phenotype Terms** fields on the **Project** editor form we need to add URL links to the help text of the field, if this ontology lives on https://bioportal.bioontology.org

**Comment by jLebioda** [comment]
The technical means to achieve having links in `help_text` is to use:

```
from django.utils.safestring import mark_safe

study_terms = models.ManyToManyField('core.StudyTerm',
                 ...
                                         help_text=mark_safe('Select one or more features that would characterize the project. Based on <a href="https://bioportal.bioontology.org/ontologies/EDDA">EDDA Study Designs Taxonomy</a>'))
```

Once I receive all the links to ontologies, I will add them to the codebase

**Comment by vildead** [comment]
Study Features - Edda - https://bioportal.bioontology.org/ontologies/EDDA
Disease Term - HDO - not in bioportal - use https://www.ebi.ac.uk/ols/ontologies/doid ? Some disease terms are already obsolete :/ see [anaplastic glioma of brain stem](http://purl.obolibrary.org/obo/DOID_7345). Also, I remember Roland being **very** clear we should use [Mondo](https://www.ebi.ac.uk/ols/ontologies/mondo) :vampire: 
Gene Terms - HGNO - only as [class](https://bioportal.bioontology.org/ontologies/HL7?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FHL7%2FC1412043) of another onthology in bioportal. Probably better to use official web - https://www.genenames.org
Phenotype names - HPO - the same, only as [class](https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C154414), better to use official - https://hpo.jax.org/

@pinarpink could you please approve the URLs?
Also, should we follow Roland's advice and create issue to include mondo instead of HDO?

**Comment by jLebioda** [comment]
Addressed in `3-url-links-in-ontology-lookup-fields` with the URLs provided by Vilem, waiting for instructions regarding Mondo

**Comment by pinarpink** [comment]
@jLebioda @vildead I donot think we can afford to do an term translation between HDO and Mondo at this very moment. Let's use below link for the HDO lookup field.
https://www.ebi.ac.uk/ols/ontologies/doid

**Comment by hcwi** [comment]
I was about to request probably the same feature that is already discussed here, so let me revive the tread.

Using the DAISY's GUI for the ontology-based terms is now difficult, if one needs to browse a flat list of unknown terms - and I guess the resulting choice is not always as precise as it could be. It would be very helpful if each field's hint gave at least the name of the terminology resource used.
